listener.go

```diff
@@ -214,6 +222,7 @@ func insertDevelopers(rows []common.Row, txn *sql.Tx) bool {
 		ele.Get("username", &Username)
 		ele.Get("first_name", &FirstName)
 		ele.Get("last_name", &LastName)
+		ele.Get("email", &Name)
```

Comment:
This is needed as a unified response for verifyApikey, where for developer Name is the email id. For company, it will be the name of the company itself.